src/applications/facility-locator/api/MockLocatorApi.js

```diff
@@ -103,6 +103,13 @@ class MockLocatorApi {
             );
           }
 
+          if (
+            locationType === 'health' &&
+            serviceType === Object.keys(healthServices)[5]
```

Comment:
Is there some reason you are referencing the `Covid19Vaccine` key this way? This should probably just be a constant.